internal/broker/bindings/bindings_manager.go

```diff
@@ -31,7 +32,7 @@ type BindingsManager interface {
 }
 
 type ClientProvider interface {
-	K8sClientSetForRuntimeID(runtimeID string) (*kubernetes.Clientset, error)
+	K8sClientSetForRuntimeID(runtimeID string) (v12.CoreV1Interface, rbac.RbacV1Interface, error)
```

Comment:
the ClientSet is a struct, not interface, so it is not possible to provide fake ClientSet

Reply:
Can we use the following interface instead of two separate interfaces: https://github.com/kubernetes/client-go/blob/235936510e4ea63c889836f7d6b06ddcf6dfa08e/kubernetes/clientset.go#L83